### PR TITLE
feat: enable colors on Windows

### PR DESCRIPTION
**Pull request by mattn**
On Windows, colors are not enabled. This change enable colors.
image
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: cmd/copilot/main.go [hunk 1]
```diff
@@ -25,16 +26,23 @@ func init() {
 	cobra.EnableCommandSorting = false // Maintain the order in which we add commands.
 }
 
-func main() {
+func run() int {
+	defer colorable.EnableColorsStdout(nil)()
```
Comment (efekarakus) [inline cmd/copilot/main.go]: I wonder why it wasn't working before 🤔 
We're using the following writers when writing to stdout:
https://github.com/aws/copilot-cli/blob/459f729744c4b3d8e4adad2e832312a2dd1d8e94/internal/pkg/term/log/log.go#L23-L24

which uses your library 🤩 as far as I understand:
https://github.com/fatih/color/blob/a05da93ebe62ca9fc6791d3376ec4dad01196448/color.go#L25-L30

Shouldn't that have been working already or am I misunderstanding how `colorable.NewColorableStdout()` works?

**Review by efekarakus — COMMENTED** [review]
Hi @mattn ! Thank you so much for the contribution, your library is awesome 😍

I just have a question below please 🙇

**Comment by efekarakus** [comment]
Side note, is it possible to update the commit like this: `feat: enable colors on Windows`? so that it makes our CI checks happy 😆

File: cmd/copilot/main.go (hunk 1 above)
Comment (mattn) [reply cmd/copilot/main.go]: Ah, copilot-cli already use go-colorable. Okay, I'll fix in another way soon.

**Comment by mattn** [comment]
I'll squash commits if you want.

**Review by efekarakus — APPROVED** [review]

**Comment by efekarakus** [comment]
> I'll squash commits if you want.

Not needed, thank you for the contribution!

**Review by Lou1415926 — APPROVED** [review]
Thank you for the contribution and your awesome library 😻

**Comment by mattn** [comment]
Thank you!

**Pull request merged**